### PR TITLE
accept receiver wallet in mint command

### PR DESCRIPTION
**Pull request by cryptorrivem**

### CODE REVIEW

File: src/mint/process.rs [hunk 1]
```diff
@@ -89,6 +90,19 @@ pub fn process_mint(args: MintArgs) -> Result<()> {
         CANDY_EMOJI
     );
 
+    let receiver_pubkey = match args.receiver {
+        Some(receiver_id) => match Pubkey::from_str(&receiver_id) {
```
Comment (samuelvanderwaal) [inline src/mint/process.rs:94]: You can make this a bit more concise by using `map_err` to convert the error from the from_str parsing error to your nice user-friendly error and then bubbling up the result with `?`:

```rust
    let receiver_pubkey = match args.receiver {
        Some(receiver_id) => Pubkey::from_str(&receiver_id)
            .map_err(|_| anyhow!("Failed to parse receiver pubkey: {}", receiver_id))?,
        None => candy_machine_state.wallet,
    };
 ```

**Review by samuelvanderwaal — COMMENTED** [review]
Thanks for the PR! It looks good, just one minor improvement and then I'll approve and merge it.

File: src/mint/process.rs (hunk 1 above)
Comment (cryptorrivem) [reply src/mint/process.rs]: replaced it with your snippet, I haven't learn much Rust, just wrote this by reading the rest of the code

Comment (samuelvanderwaal) [reply src/mint/process.rs]: All good, just wanted to give you a little tip and keep the code a little more maintainable.

**Review by samuelvanderwaal — APPROVED** [review]

**Pull request merged**